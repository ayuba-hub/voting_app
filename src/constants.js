//Replace the "constants.js" with this in order to run on MainNet
const ASSET_ID = 297995609;
const ADDRESS_1 = "Q7Y4STQ3ULFRT6ZSBHRKVEVKZOU7JKOIEZH5V4BDBEWUZ3OVNCSFFKG2V4";
const ADDRESS_2 = "LONCFPK6NP7SFDGGULOWTRROHY5THPX73RSZQF5I5LASRC5JASCN6NUVMM";
const URL = "https://choice-coin-api.herokuapp.com";
const ELECTION_ID = "9c365976-8477-4ee8-a960-c0cdf8624bc";
export { ASSET_ID, URL, ELECTION_ID, ADDRESS_1, ADDRESS_2 };
