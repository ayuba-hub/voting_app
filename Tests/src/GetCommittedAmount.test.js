import {shallow} from 'enzyme';
import React from 'react';
import GetCommittedAmount from './GetCommittedAmount';
import * as redux from 'react-redux'
import {mockLocalStorage} from './LocalStorageMock';
import axios from 'axios';
import MockAdapter from 'axios-mock-adapter';
import { URL } from "./constants";

const { getItemMock, setItemMock } = mockLocalStorage();

describe("test on GetCommittedAmount Component", ()=> {
 

    const spy = jest.spyOn(redux, 'useSelector')
    spy.mockReturnValue({ AddressNum: 1 })

  

    it('expects to render GetCommittedAmount Component', ()=> {
        getItemMock.mockReturnValue('address');
        expect(setItemMock).not.toHaveBeenCalled();
       expect(shallow(<GetCommittedAmount/>)).toMatchSnapshot()
    })


    it('expects to render GetCommittedAmount Component', () => {
        getItemMock.mockReturnValue('address');
        expect(setItemMock).not.toHaveBeenCalled();
        // const wrapper = shallow(<GetCommittedAmount/>)
        // wrapper.find('[id ="amt"]')
        expect(shallow(<GetCommittedAmount/>).length).toEqual(1)
    })
})