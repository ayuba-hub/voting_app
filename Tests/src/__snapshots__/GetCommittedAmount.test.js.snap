// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test on GetCommittedAmount Component expects to render GetCommittedAmount Component 1`] = `ShallowWrapper {}`;
